models/intermediate/int_quickbooks__account_classifications.sql

```diff
@@ -3,6 +3,12 @@ with accounts as (
     from {{ ref('stg_quickbooks__account') }}
 ),
 
+parent_accounts as (
+    select *
+    from accounts
+    where not is_sub_account
```

Comment:
is it possible to be a sub_account and a parent account at the same time?

For example, child --> parent --> grandparent.  here, parent would be both. I'm not sure if this is possible in Quickbooks but I believe this could happen in Netsuite.

Reply:
Updated the logic to simply join account on itself on `parent_account_id`